.github/workflows/test_colcon.meta

```diff
@@ -15,7 +15,7 @@
                 "-DBUILD_DOCUMENTATION=ON",
                 "-DBUILD_TESTS=ON",
                 "-DCMAKE_BUILD_TYPE=Debug",
-                "-DCMAKE_EXPORT_COMPILE_COMMANDS=ON",
```

Comment:
I think that removing this line is preventing the coverage job to run.

Reply:
Not sure about it.
Added in commit 302cb09 to check.

Reply:
Let us see if it works now